ovs/lib/scheduledtask.py

```diff
@@ -191,6 +196,8 @@ def make_timestamp(offset):
         for bucket_chain in bucket_chains:
             for bucket in bucket_chain:
                 for snapshot in bucket['snapshots']:
+                    if snapshot.get('is_sticky') is True:
+                        continue
```

Comment:
There shouldn't be any sticky snapshots in here, based on the filters above, right?